src/main/java/org/folio/circulation/domain/Item.java

```diff
@@ -178,7 +198,14 @@ public String getMaterialTypeName() {
   }
 
   public JsonArray getCopyNumbers() {
-    return getArrayProperty(getItem(), "copyNumbers");
+    JsonArray copyNumbers = getArrayProperty(getItem(), "copyNumbers");
+    return getEffectiveCopyNumbers(copyNumbers);
+  }
+
+  private JsonArray getEffectiveCopyNumbers(JsonArray copyNumbers) {
+    return copyNumbers.isEmpty()
```

Comment:
What behaviour is this trying to describe, that the item copy numbers takes precedence over the holdings record definition?

Reply:
Yes, that's what I mean. The copy numbers of item record take precedence over the holdings record definition